packages/synapse-react-client/src/components/widgets/query-filter/FacetFilterControls.tsx

```diff
@@ -221,6 +221,25 @@ function FacetFilterControls(props: FacetFilterControlsProps) {
 
   return (
     <div className={`FacetFilterControls`}>
+      <div>
```

Comment:
Moved up.  Only changed the header label (to Available Filters)

Reply:
also an elegant way to fix an annoying layout shift issue, where filter sections would appear/disappear as each chip was clicked!